geotrek/maintenance/forms.py

```diff
@@ -29,6 +30,10 @@ def __init__(self, *args, **kwargs):
         self.fields['nb_days'].label = ''
         self.fields['nb_days'].widget.attrs['class'] = 'input-mini'
         self.fields['job'].widget.attrs['class'] = 'input-medium'
+        if self.instance and self.instance.pk:
+            self.fields['job'].queryset = InterventionJob.objects.filter(Q(active=True) | Q(id=self.instance.job.id))
```

Comment:
```suggestion
            self.fields['job'].queryset = InterventionJob.objects.filter(Q(active=True) | Q(id=self.instance.job_id))
```
chaque foreign key a son attribut _id par defaut. en l'utilisant tu evites à django de faire une requete sql inutile